### DISCUSSION TITLE
What is the purpose of module_basis/module_nao?

### DISCUSSION THREAD

**Discussion by Satinelamp** [deepmodeling/abacus-develop#2522] [Q&A] [answered]
I am wondering why there is a new module_nao. Is it for refactoring module_ao?

**Comment by jinzx10** [comment, marked as answer]
Yes, it is supposed to do similar things as module_ao do with a more structured code design. Apart from refactoring module_ao, it'll also employ more a robust algorithm for two-center integrals.
